frb_codegen/src/ir/ty.rs

```diff
@@ -66,6 +66,11 @@ impl IrType {
         matches!(self, StructRef(_) | EnumRef(_))
     }
 
+    #[inline]
+    pub fn is_optional(&self) -> bool {
+        matches!(self, Optional(_))
+    }
+
```

Comment:
nit: Looks like this change can be reverted since not used?

Reply:
reverted in 61b435a